src/SamplesApp/SamplesApp.UnitTests.Shared/Controls/UITests/Views/Controls/SampleChooserControl.xaml

```diff
@@ -530,6 +531,12 @@
 							  Grid.Column="8"
 							  IsChecked="{Binding UseFluentStyles, Mode=TwoWay}"
 							  Style="{x:Null}"/>
+					<!-- Dark theme check box -->
+					<CheckBox Grid.Column="9"
+							  IsChecked="{Binding UseDarkTheme, Mode=TwoWay}"
+							  Style="{x:Null}">
```

Comment:
nit: You probably don't need `Style="{x:Null}"` here :) (but shouldn't do any harm either)

IIRC for the Fluent check box it was to force it to always use the UWP style, because there was some sort of weirdness (involving visual states?) if its style changed inside of its own checked callback.